src/modules/Kdump.rb

```diff
@@ -1145,7 +1145,11 @@ def crash_kernel_values
       # modification.
       # The old value (ensuring the Array format) will be returned.
       if @crashkernel_list_ranges
-        return Array(@crashkernel_param_values.dup)
+        if @crashkernel_param_values.is_a? Symbol
+          return @crashkernel_param_values
+        else
+          return Array(@crashkernel_param_values.dup)
+        end
```

Comment:
I think this is wrong. `#crash_kernel_values` is expected, as it documentation states, to return an array, which now is not always true.

Since this method is documented to return `[Array<String>]`, I believe a better fix would be
```ruby
if @crashkernel_list_ranges
  return Array(@crashkernel_param_values.to_s.dup)
end
```
That would work consistently for strings and symbols.